gnocchi/storage/ceph.py

```diff
@@ -153,10 +153,8 @@ def _get_measures_unbatched(self, metric, key, aggregation, version=3):
         except rados.ObjectNotFound:
             if self._object_exists(
                     self._build_unaggregated_timeserie_path(metric, 3)):
```

Comment:
do we still need these checks? if ObjectNotFound, isn't it a given that it should raise MetricDoesNotExist?

Reply:
No, it just means there's no data for this particular split. When a split is rewritten, the storage engine tries to read it and if data are there it updates. But you're right that I think we can simplify this again.

If that's ok I'll do that in a subsequent patch. I saw some things that we can simplify with that.